hrjobcontract/CRM/Hrjobcontract/Upgrader.php

```diff
@@ -1252,6 +1253,43 @@ public function upgrade_1036() {
     return TRUE;
   }
 
+  /**
+   * Create new option group and values
+   */
+  public function upgrade_1037() {
+    $groupValues = [
+      'As per contract',
+      'Change in contractual hours',
+      'Change in contract type'
+    ];
+
+    $result = civicrm_api3('OptionGroup', 'get', [
+      'name' => 'hrleaveandabsences_work_pattern_change_reason'
+    ]);
+
+    if ($result['count'] == 0) {
```

Comment:
How about changing this to use an early return, like

```
if ($result['count'] > 0) {
  return TRUE;
}

// create the group
```

This would avoid an extra level of indentation for most of the function